src/components/Address.vue

```diff
@@ -1,6 +1,14 @@
 <template>
-	<span :title="email"
-		  v-tooltip.bottom="label">{{ label }}</span>
+	<router-link :to="{name: 'message', params: {
+								accountId: $route.params.accountId,
```

Comment:
Too much tabs? :p

Reply:
![bildschirmfoto von 2018-11-21 18-47-43](https://user-images.githubusercontent.com/1374172/48859275-f338fc00-edbd-11e8-8096-7e04bd499cee.png)
works on my machine :stuck_out_tongue:

Reply:
I should move that to a computed prop. it's ugly in the template.

Reply:
> works on my machine :stuck_out_tongue:

We have very different coding style ^^'

Reply:
Yeah. Please use mine, it's so much better :wink: